mesop/components/input/e2e/input_blur_app.py

```diff
@@ -0,0 +1,42 @@
+import mesop as me
+
+
+@me.stateclass
+class State:
+  input: str = ""
+  input_value_when_button_clicked: str
+
+
+def on_input(e: me.InputEvent):
+  state = me.state(State)
+  state.input = e.value
+
+
+def on_enter(e: me.EnterEvent):
+  state = me.state(State)
+  state.input = "boo"
```

Comment:
[question] What's this boo about? 

The on blur won't work with on enter right?

Reply:
oops, this was a remnant from input_app.py, removed.

yeah, on_blur and on_enter don't work well, I'm thinking about changing on_enter to also include the input value